### PR TITLE
disabling performance tests

### PR DESCRIPTION
**Pull request by dbulahov**
Signed-off-by: dbulahov <bulahovdenis@gmail.com>

### CODE REVIEW

File: olap/rolap/dbmapper/mondrian/src/test/java/org/eclipse/daanse/olap/rolap/dbmapper/mondrian/WrongSchemaTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import static org.eclipse.daanse.olap.rolap.dbmapper.mondrian.ReadTest.extracted;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class WrongSchemaTest {
+
+    @Test
+    void testRrequiredAatrebutesInWrongSchema() throws Exception {
+        try {
+            Schema schema = extracted( new File("./src/test/resources/WrongFoodMart.xml"));
```
Comment (stbischof) [inline olap/rolap/dbmapper/mondrian/src/test/java/org/eclipse/daanse/olap/rolap/dbmapper/mondrian/WrongSchemaTest.java:30]: this test just runs via normal junit5 or also with full integration test?

File: olap/rolap/dbmapper/mondrian/src/test/resources/WrongFoodMart.xml [hunk 2]
```diff
@@ -0,0 +1,827 @@
+<?xml version="1.0"?>
+<Schema name="FoodMart">
```
Comment (stbischof) [inline olap/rolap/dbmapper/mondrian/src/test/resources/WrongFoodMart.xml:2]: could you use a much simpler example that provicates the validation error?

File: olap/rolap/dbmapper/mondrian/src/test/java/org/eclipse/daanse/olap/rolap/dbmapper/mondrian/WrongSchemaTest.java (hunk 1 above)
Comment (dbulahov) [reply olap/rolap/dbmapper/mondrian/src/test/java/org/eclipse/daanse/olap/rolap/dbmapper/mondrian/WrongSchemaTest.java:30]: run as junit5

File: olap/rolap/dbmapper/mondrian/src/test/resources/WrongFoodMart.xml (hunk 2 above)
Comment (dbulahov) [reply olap/rolap/dbmapper/mondrian/src/test/resources/WrongFoodMart.xml:2]: Yes. I plan to do that when validation will work

**Pull request merged**